### PR TITLE
[IMP] tools: Display performance time in thread stack dump

### PR DESCRIPTION
**Pull request by lse-odoo**
Before this commit:
 Thread details gave information regarding the database
 the UID and the URL.

After this commit:
  It also informs on the "performance times":
  - `qc` = query count
  - `qt` = query time
  - `pt` = python time. It should in theory be "remaining time", but it won't be use as its acronym `rt` is too ambiguous/confusing

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by lse-odoo** [comment]
the main reason behind this PR is in the context of the support to troubleshoot odoo.sh issues on which some process are very long. However when we do send the signal 3 (which cause a `dumpstacks`) , we can't easily say the time took by each thread. It we had this information, it can help us find execution bottlenecks on most of the cases.

**Comment by lse-odoo** [comment]
@odony May I have your opinion on this PR please ? :)

**Comment by lse-odoo** [comment]
> Sounds like a good idea :)
> 
> It won't display anything relevant for non-http threads, but that's fine.

I think it is indeed time to use clear acronyms rather than losing time each time wondering which value correspond to which. Thank you for the suggestions :) Do you think someone else in particular should review this PR ?

**Review by Julien00859 — APPROVED** [review]

**Comment by lse-odoo** [comment]
@odony Would you mind approving in on the behalf of the security team if it looks correct to you ? I don't really see who should review this PR

**Pull request closed without merging**
